src/App.jsx

```diff
@@ -37,24 +39,24 @@ function App() {
     console.log(fetchData());
   }, []);
 
+  const generateNewArt = async () => {
```

Comment:
also moved this to a helped function to be consumed in a more readable format in the rendering of this component